### PR TITLE
Video controls: never hide when paused

### PR DESCRIPTION
**Pull request by dcposch**
Also simplifies CSS

### CODE REVIEW

**Comment by ChrisMorrisOrg** [comment]
Oops, I was just working on this myself. I did something along the lines of:

/renderer/lib/display-controls.js
```
module.exports = displayControls

// When in fullscreen, hide player controls if the mouse stays still for a while
function displayControls() {
  if (state.video.mouseStationarySince !== 0 && new Date().getTime() - state.video.mouseStationarySince < 2000) {
    return true
  }
  if (state.video.isPaused) {
    return true
  }
  return false
}
```

Not sure what the best way of handling the seek bar drag event, though.

Your code is perfectly fine as well! :)

File: renderer/views/app.js [hunk 1]
```diff
@@ -11,10 +11,21 @@ var TorrentList = require('./torrent-list')
...
+    state.video.mouseStationarySince !== 0 &&
+    new Date().getTime() - state.video.mouseStationarySince > 2000 &&
+    !state.video.isPaused
+
   var cls = [
-    process.platform,
+    process.platform, /* 'windows', 'linux', or 'darwin' for OSX */
     state.isFullScreen ? 'fullscreen' : 'not-fullscreen',
-    state.url === '/player' ? 'view-player' : ''
+    isVideoPlayer ? 'view-player' : ''
```
Comment (rom1504) [inline renderer/views/app.js]: you're missing a comma here. travis is unhappy about it

**Comment by dcposch** [comment]
@ChrisMorrisOrg it looks like dragging keeps `.player-controls` in the hover state, even when the mouse is not over the player controls!

When I hold the mouse still for >2 seconds while scrubbing, the player controls still don't disappear --- so I think we're good

File: renderer/views/app.js (hunk 1 above)
Comment (dcposch) [inline renderer/views/app.js]: good call, fixed

**Comment by dcposch** [comment]
@ChrisMorrisOrg sorry about duplicating work, didn't know

**Comment by dcposch** [comment]
Travis is trippin

```
$ npm test

> webtorrent-app@0.0.0 test /home/dc/webtorrent-app
> standard

$
```

**Comment by ChrisMorrisOrg** [comment]
@dcposch no worries :), the CSS solution is better anyway!

**Pull request merged**